src/modules/components/vote/voteProposalDataListItem/voteProposalDataListItemStructure/voteProposalDataListItemStructure.tsx

```diff
@@ -20,10 +20,14 @@ export type IVoteProposalDataListItemStructureProps = IDataListItemProps & {
      *  Date of the vote on the proposal in ISO format or as a timestamp
      */
     date?: number | string;
+    /**
+     *   Custom label for the tag
+     */
+    voteLabel?: string;
```

Comment:
I would say we call this something like `confirmationLabel` as it describes the status. "Voted" "Voting" "Will vote" etc and update doc line to include this idea. 

We also generally try not to use the main intent of the component name in the interface. I think it's a good exercise in a SOLID kind of way.